dCommon/dEnums/eInventoryType.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#ifndef __EINVENTORYTYPE__H__
+#define __EINVENTORYTYPE__H__
+
+#include <cstdint>
+static const uint8_t NUMBER_OF_INVENTORIES = 17;
+/**
+ * Represents the different types of inventories an entity may have
+ */
+enum eInventoryType : uint32_t {
+	ITEMS = 0,
+	VAULT_ITEMS,
+	BRICKS,
+	MODELS_IN_BBB,
+	TEMP_ITEMS,
+	MODELS,
+	TEMP_MODELS,
+	BEHAVIORS,
+	PROPERTY_DEEDS,
+	BRICKINBBB,
```

Comment:
`BRICKS_IN_BBB` to match the above `MODELS_IN_BBB`

Reply:
sounds good